pkgs/tools/networking/isync/default.nix

```
@@ -6,22 +6,17 @@
 , withCyrusSaslXoauth2 ? false, cyrus-sasl-xoauth2, makeWrapper
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "isync";
-  version = "1.4.4";
+  version = "1.5.0";
 
   src = fetchurl {
-    url = "mirror://sourceforge/isync/${pname}-${version}.tar.gz";
-    sha256 = "1zq0wwvmqsl9y71546dr0aygzn9gjjfiw19hlcq87s929y4p6ckw";
+    url = "mirror://sourceforge/isync/${finalAttrs.pname}-${finalAttrs.version}.tar.gz";
```

```suggestion
    url = "mirror://sourceforge/isync/isync-${finalAttrs.version}.tar.gz";
```

Using `finalAttrs.pname` here is a bad idea.  If `pname` is overridden like this, this url becomes invalid.

```nix
isync.overrideAttrs (old: { pname = old.pname + "-patched"; })
```

I think it is better to use `pname` with `rec` or just hardcode `isync`.